Makefile

```diff
@@ -67,28 +68,33 @@ $(EXAMPLE_TARGETS): example-%:
 all: check test build binary
 check: $(CHECK_TARGETS)
 
-# Update the vendor folder
+## vendor: Update the vendor folder
 vendor:
 	go mod vendor
 
-# Apply go fmt to the codebase
+## fmt: Apply go fmt to the codebase
 fmt:
 	go list -f '{{.Dir}}' $(MODULE)/... \
 		| xargs gofmt -s -l -w
 
+## golangci-lint: Run the golangci-lint tool
 golangci-lint:
-	golangci-lint run ./...
+	golangci-lint run ./... --fix
```

Comment:
Since `golangci-lint` is run from CI, does it make sense to have the command with `--fix` as a separate target?

Regardless, I would prefer to have this change in a different PR.